content/blog/a-guide-to-enable-managed-istio-service-mesh-in-a-kubernetes-cluster-on-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,214 @@
+
+**Step-1: Create a Kubernetes cluster from containers page**
+
+To create a cluster, you must have been assigned the roles of **Private Cloud Cluster Owner** and **Private Cloud Widget Viewer**.
+
+* From the Containers main page, under the Clusters tab, click Create Cluster.
+* In the Create Cluster form, provide the cluster name '**hpe**', and select the standard cluster blueprint. The new cluster appears in the list of clusters.
+
+![](/img/clustermainpage-2.png)
+
+As indicated above, there are multiple clusters deployed in parallel for multiple purposes. For the **Istio** service mesh add-on enablement/deployment in our example, we will use created cluster with the name '**hpe**'.
+
+![](/img/clusterhpeview.png)
+
+**Step-2: Enabling an add-on from a cluster**
+
+* On the Containers main page, click a cluster row to open the cluster details screen.
+* On the cluster details screen, click the Add-ons tab.
+* Click Enable Add-on if no add-ons are enabled or click Enable Another Add-on.
+
+![](/img/blankaddonpage.png)
+
+* In the Enable Addon wizard, select the add-on and click Next.
+
+![](/img/istioaddonpage-11.png)
+
+* Provide the values for the fields that appear for the selected add-on, read and accept the license agreement, and click Enable.
+
+![](/img/istioaddonpage-22.png)
+
+* A﻿fter successful add-on enablement, add-on status will get updated to '**succeeded**'.
+
+![](/img/istioaddongreenstatus.png)
+
+* V﻿iew the details of the add-on that you just enabled.
+
+![](/img/istioaddondetailspage.png)
+
+**Step-3: Launching the Kiali dashboard**
+
+From the Overview tab, click the **KialiURL** link and use the **Kiali token**.
+
+The **Kiali dashboard** launches in a new web page.
+
+**Note**: KialiURL might be different in your environment.
+
+![](/img/kiali-console.png)
+
+**Note**: To take advantage of the Istio features, pods in the mesh must be running an Istio sidecar proxy. Injection of the proxy can be done either on a per-pod basis or at namespace level. To enable side car injection, see **<https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/>**. For information about using Kiali, see the **[Kiali documentation](https://kiali.io/docs/)**.
+
+**Step-4: Download scoped kubeconfig from Container platform page**
+
+* From the Clusters tab, select the '**hpe**' Kubernetes cluster and click **Launch Service Console**. This will direct you to the container platform page.
+* Click on Download **kubeconfig**.
+
+**Note**: Launching HPE Ezmeral Runtime Enterprise from HPE GreenLake Central is configured through SAML SSO and adds a session token to the kubeconfig file. You will need to download the kubeconfig file again if you want to continue to access the cluster when the session token expires after an hour.
```

Comment:
Is it necessary to  mention "HPE Ezmeral Runtime Enterprise" here?
Maybe something like this could make the trick:

The kubeconfig file stores cluster authentication information. It is configured with a secure and time-limited (60 minutes) session token. You will need to download the kubeconfig file again if you want .....

Reply:
done

updated like this;

**Note**: Launching service console from HPE GreenLake Central is configured through SAML SSO and adds a session token to the kubeconfig file. You will need to download the kubeconfig file again if you want to continue to access the cluster when the session token expires after an hour.

Removed mention of HPE Ezmeral Runtime Enterprise